docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -8,6 +8,12 @@ An Expectation is a verifiable assertion about your data. They make implicit ass
 
 <!-- [//]: # (TODO: To learn more about Expectations, see Expectation.) -->
 
+:::info Custom SQL Query Expectations
+
+Support for custom SQL query Expectations is currently unavailable in GX Cloud. To create custom SQL query Expectations, see [Customize an Expectation Class](/core/create_expectations/expectations/manage_expectations.md#customize-an-expectation-class).
```

Comment:
I might phrase this differently. This note is about the inability to create custom SQL expectations from the Cloud UI. If you create a custom SQL expectation via the API you can still view and reference it in the Cloud UI, just as you can with the 200+ expectations in the Gallery that are not available in Cloud. Is "unavailable" the right word?

Reply:
@annabarr we _might_ be splitting hairs! :)

The statement, as written, is true. Custom SQL query Expectations are currently unavailable in the GX Cloud UI. Users need to use the GX API to configure them. This cannot be done in GX Cloud. Therefore, the statement as written is accurate. 

If you prefer to soften the text, here is an alternate version:

_To create custom SQL query Expectations, you'll need to use the GX API. See [Customize an Expectation Class](/core/create_expectations/expectations/manage_expectations.md#customize-an-expectation-class)._

The revised text will not appear in a note admonition, but will be added after the opening statement.

Reply:
I like that better :)

Reply:
Sold! I'll make the change and merge the change.